app/Http/Transformers/SignupTransformer.php

```diff
@@ -26,10 +26,12 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
+        $signup_event = Event::where('id', $signup->event_id)->get();
+
         return [
             'signup_id' => $signup->id,
             'signup_event_id' => $signup->event_id,
-            'submission_type' => $signup->event->submission_type,
+            'submission_type' => $signup_event[0]->submission_type,
```

Comment:
You should be able to just grab the first item from the relationship, rather than re-querying. 

```php
$signup->events->first()->submission_type
```